src/live.asp.net/ViewModels/HomeViewModel.cs

```diff
@@ -28,5 +28,21 @@ public class HomeViewModel
         public string MoreShowsUrl { get; set; }
 
         public bool ShowMoreShowsUrl => !string.IsNullOrEmpty(MoreShowsUrl);
+
+        public string AddToGoogleUrl
+        {
+            get
+            {
+                // reference: http://stackoverflow.com/a/21653600/22941
+
+                const string eventTitle = "ASP.NET Community Standup";
```

Comment:
We only use `const` for actual constant things that will never ever change. Granted, in an application it doesn't make much difference as nothing will be referencing it, but I'd like to keep the coding style consistent.